composer.json

```diff
@@ -27,13 +27,13 @@
         "cocur/slugify": "^1.4 || ^2.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-config-test": "~0.4 || ^1.0",
-        "matthiasnoback/symfony-dependency-injection-test": "~0.7",
+        "matthiasnoback/symfony-config-test": "^0.4 || ^1.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7",
         "doctrine/orm": "^2.4",
         "doctrine/phpcr-odm": "^1.0",
         "jackalope/jackalope-doctrine-dbal": "^1.0",
         "friendsofsymfony/rest-bundle": "^1.1 || ^2.0",
-        "jms/serializer-bundle": "~0.11 || ^1.0",
+        "jms/serializer-bundle": "^0.11 || ^1.0",
```

Comment:
>> if we would use ^0.11 || ^0.12 || ^0.13 did we run all the tests in all combinations?
>
> What do you mean?

I mean does travis run all possible combinations or do we trust the maintainer? i mean, how to be sure, that `0.12` is working ?